assignments/urls.py

```diff
@@ -12,5 +11,6 @@
 api_urls = router.urls
 
 urlpatterns = [
-    url(r'^api/', include(api_urls))
+    url(r'^api/', include(api_urls)),
+    url(r'^api/answers/(?P<slug>[-\w]+)/$', SubmitAnswersView.as_view(), name='answers')
```

Comment:
It is usually a good idea to add a comma at the end of lists. This way you don't get the same stuff happening in as here when you remove and add a new line in the diff first, and then you add the actual line that is new.